File: RentalAttireFrontend/src/app/features/admin/purchase-orders/dtos/purchase-order-dto.ts
import { PurchaseOrderItemDTO } from "./purchase-order-item-dto";

export interface PurchaseOrderDTO {
    id : number;
    purchaseOrderCode : string;
    orderDate : Date;
    expectedDeliveryDate : Date;
    supplierName : string;
    orderStatus : string;
    employeeName : string;
    totalAmount : number;
    purchaseOrderItems : PurchaseOrderItemDTO[];
}

File: RentalAttireBackend/RentalAttireBackend/Application/PurchaseOrders/DTOs/PurchaseOrderDTO.cs
using RentalAttireBackend.Application.PurchaseOrderItems.DTOs;

namespace RentalAttireBackend.Application.PurchaseOrders.DTOs
{
    public class PurchaseOrderDTO
    {
        public int Id { get; set; }
        public string PurchaseOrderCode { get; set; } = string.Empty;
        public DateTime OrderDate { get; set; } 
        public DateTime ExpectedDeliveryDate { get; set; }
        public string SupplierName { get; set; } = string.Empty;
        public string OrderStatus { get; set; } = string.Empty;
        public string EmployeeName { get; set; } = string.Empty;
        public double TotalAmount { get; set; }
        public List<PurchaseOrderItemDTO> PurchaseOrderItems { get; set; } = new();
    }
}
